src/FourteenDayActiveUsersMetric.php

```diff
@@ -0,0 +1,142 @@
+<?php
```

Comment:
It feels like these metrics are so similar we could potentially have all of them extend a single class, or use a trait, or something?